src/test/java/ProfileTests.java

```diff
@@ -0,0 +1,15 @@
+import org.testng.annotations.Test;
+import pages.LoginPage;
+import pages.ProfilePage;
+
+public class ProfileTests extends BaseTest {
+@Test
+public void differentBackground() throws InterruptedException {
+    LoginPage loginPage = new LoginPage(getDriver());
+    ProfilePage profilepage = new ProfilePage(getDriver());
+    loginPage.login();
+    Thread.sleep(2000);
+    profilepage.avatarIconBtn();
+    profilepage.chooseVioletTheme();
+}
+}
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)